rubrik_cdm/data_management.py

```diff
@@ -302,11 +312,20 @@ def object_id(self, object_name, object_type, host_os=None, timeout=15):
             else:
                 name_value = 'name'
 
+            host_match = False
```

Comment:
I would put `host_match = False` under `if object_type == 'oracle_db' and item[name_value] == object_name:` that way it's only called if it needs to be.